src/components/FeeLabelToggle/FeeLabelToggle.js

```diff
@@ -93,21 +92,20 @@ const FeeLabelToggle = ({
   rates,
   isLoading,
   labelText,
-  showFiatDefault,
   showRelayerMigration = true,
   accountAssets,
   accountHistory,
   isGasTokenSupported,
   hasError,
 }: Props) => {
-  const [isFiatValueVisible, setIsFiatValueVisible] = useState(showFiatDefault);
+  const [isFiatValueVisible, setIsFiatValueVisible] = useState(true);
 
   if (isLoading) {
     return <Spinner size={20} trackWidth={2} />;
   }
 
   const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
-  const feeDisplayValue = formatTransactionFee(txFeeInWei, gasToken);
+  const feeDisplayValue = getFormattedTransactionFeeValue(txFeeInWei, gasToken);
```

Comment:
This is incorrect as this causes the FeeLabelToggle to display amount without token. You need two separate variables:
`const feeDisplayValue = formatTransactionFee(txFeeInWei, gasToken);` for display label
`const feeValue = getFormattedTransactionFeeValue(txFeeInWei, gasToken);` for `parseFloat` input.